tests/foreman/ui/test_settings.py

```diff
@@ -489,3 +490,42 @@ def test_negative_update_hostname_with_empty_fact(session):
         finally:
             for key, value in default_hostname.items():
                 setting_cleanup(setting_name=key, setting_value=value.value)
+
+
+@run_in_one_thread
+@tier3
+def test_positive_entries_per_page(session):
+    """ Update the per page entry in the settings.
+
+    :id: 009026b6-7550-40aa-9f78-5eb7f7e3800f
+
+    :Steps:
+        1. Navigate to Administer > Settings > General tab
+        2. Update the entries per page value
+        3. GoTo Monitor > Tasks Table > Pagination
+        4. Check the new per page entry is updated in pagination list
+        5. Check the page count on the basis of the new updated entries per page.
+
+    :expectedresults: New set entry-per-page should be available in the pagination list and
+        page count should match according to the new setting
+
+    :CaseImportance: Medium
+
+    :CaseLevel: Acceptance
```

Comment:
Can we please add the Bugzilla ID here for future reference

Reply:
Fixed